test/PushNotificationTests/APITests.cpp

```diff
@@ -24,10 +24,12 @@ namespace Test::PushNotifications
     {
     private:
         wil::unique_event m_failed;
+        HANDLE m_process;
```

Comment:
No need to use raw pointers.

```suggestion
        wil::unique_handle m_process;
```